test/test_playbooks/content_view_filter.yml

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
Is this the beginning of test driven development?

Reply:
:man_facepalming: